tests/tests/redis/mod.rs

```diff
@@ -12,22 +12,22 @@ pub async fn assert_bytes(cmd: &mut Cmd, connection: &mut MultiplexedConnection,
     assert_eq!(cmd.query_async(connection).await, Ok(value.to_vec()));
 }
 
-pub async fn redis_cluster_connection(address: &str) -> redis::cluster_async::ClusterConnection {
+pub async fn valkey_cluster_connection(address: &str) -> redis::cluster_async::ClusterConnection {
     let client = ClusterClient::new(vec![address]).unwrap();
     client.get_async_connection().await.unwrap()
 }
 
-pub async fn redis_single_connection(address: &str) -> redis::aio::MultiplexedConnection {
+pub async fn valkey_single_connection(address: &str) -> redis::aio::MultiplexedConnection {
     let client = redis::Client::open(address).unwrap();
     client.get_multiplexed_tokio_connection().await.unwrap()
 }
 
 #[tokio::test(flavor = "multi_thread")]
 #[serial]
-async fn redis_cluster_1_1() {
+async fn valkey_cluster_1_1() {
     let _compose = docker_compose("../redis-cluster-1-1/docker-compose.yaml");
 
-    let mut connection = redis_cluster_connection("redis://172.16.1.2:6380").await;
+    let mut connection = valkey_cluster_connection("redis://172.16.1.2:6380").await;
```

Comment:
Should this be `valkey:`?

Reply:
The driver doesnt support valkey prefix yet